Gems/PhysX/Code/Source/EditorColliderComponent.h

```diff
@@ -197,6 +198,9 @@ namespace PhysX
         void SetAssetScale(const AZ::Vector3& scale) override;
         AZ::Vector3 GetAssetScale() override;
 
+        // PhysX::EditorColliderValidationRequestBus
```

Comment:
Nit: if this is the impl if that bus the comment should read `// PhysX::EditorColliderValidationRequestBus overrides ...`.